File: scripts/check-products.mjs
import { readFileSync, writeFileSync } from 'fs';
import { execSync } from 'child_process';

// Check suppliers
console.log('=== SUPPLIERS ===');
const supRaw = execSync('npx firebase database:get /suppliers --project house-menuapp', { encoding: 'utf8', timeout: 15000 });
const suppliers = JSON.parse(supRaw);
for (const [id, s] of Object.entries(suppliers)) {
  console.log(`${id}: ${s.name} (active: ${s.active})`);
}

// Check products
console.log('\n=== PRODUCTS (monteverde) ===');
const prodRaw = execSync('npx firebase database:get /branches/monteverde/catalog/products --project house-menuapp', { encoding: 'utf8', timeout: 15000 });
const products = JSON.parse(prodRaw);
for (const [id, p] of Object.entries(products).sort((a,b)=>a[1].name?.localeCompare(b[1].name))) {
  console.log(`${id}: ${p.name} | minStock=${p.minStock ?? '-'} | supplierId=${p.supplierId ?? '-'} | stock=${p.stock ?? '-'} | trackStock=${p.trackStock ?? '-'}`);
}

File: scripts/check_products.py
import json, os

path = os.environ['TEMP'] + '\\products.json'
with open(path) as f:
    d = json.load(f)

if not d:
    print('No products found at /branches/monteverde/catalog/products')
else:
    for k, v in sorted(d.items(), key=lambda x: x[1].get('name', '')):
        name = v.get('name', '?')
        min_stock = v.get('minStock', '-')
        supplier = v.get('supplierId', '-')
        stock = v.get('stock', '-')
        track = v.get('trackStock', '-')
        avail = v.get('available', True)
        print(f'{k}: {name} | minStock={min_stock} | supplierId={supplier} | stock={stock} | trackStock={track} | available={avail}')
